src/index.tsx

```diff
@@ -1,8 +1,3 @@
 import { render } from 'react-dom';
-
 import App from './App';
-import { unregister } from './registerServiceWorker';
```

Comment:
I'm pretty sure this is purely an artifact of CRA, and not relevant in Vite

Reply:
Yeah, confirmed with network inspector that removing this doesn't stop the page from caching itself.